_includes/footer.html

```diff
@@ -13,7 +18,7 @@
     {%- else -%}
     <footer class="sticky-bottom mt-5">
       <div class="container">
-        &copy; Copyright {{ site.time | date: '%Y' }} {% t main.first_name %} {% t main.middle_name %} {% t main.last_name %}. {% t main.footer_text %}
+        &copy; Copyright {{ name_components | split:' ' | join:' ' }}. {% t main.footer_text %}
```

Comment:
Same here, keep the year

Reply:
Right, should have read all your comments before committing. Resolved with 2f16533.